totest-manager.py

```diff
@@ -384,7 +384,7 @@ def is_snapshottable(self):
 
                 for arch in ['i586', 'x86_64']:
                     for product in self.livecd_products:
-                        if not self.package_ok('openSUSE:%s:Live' % self.project, product, 'standard', arch):
+                        if not self.package_ok('openSUSE:%s:Live' % self.project, product, 'images', arch):
```

Comment:
This would appear to not work for the multibuild packages:

```
osc api '/build/openSUSE:Factory/_result?package=livecd-openSUSE:livecd-kde&repository=images&arch=x86_64'
Server returned an error: HTTP Error 404: unknown package 'livecd-openSUSE livecd-kde'
unknown package 'livecd-openSUSE:livecd-kde'
```

Reply:
Package does not exist in `Factory`, but does in `Factory:Live`.